students/BrianE/lesson03/tests/test_gradel03.py

```diff
@@ -0,0 +1,173 @@
+    Run cobverage and linitng using standard batch file
+    Student should submit an empty database
+
+"""
+
+import pytest
+from peewee import *
+
+import customer_data.basic_operations as l
+
+database = SqliteDatabase("customers.db")
+database.connect()
+database.execute_sql("PRAGMA foreign_keys=ON;")
+database.execute_sql('drop table if exists customer;')
+l.Customer.create_table()
+
+@pytest.fixture
+def _add_customers():
+    return [
+        ("123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("456", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("789", "Name", "Lastname", "Address", "phone", "email", "active", 0),
+        ("345", "Name", "Lastname", "Address", "phone", "email", "active", -10),
+        ("0123", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("777", "Name", "Lastname", "Address", "phone", "email", "active", 999)
+    ]
+
+@pytest.fixture
+def _search_customers(): # needs to del with database
+    return [
+        [("998", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+         ("997", "Name", "Lastname", "Address", "phone", "email", "inactive", 10)],
+        ("998", "000")
+    ]
+@pytest.fixture
+def _delete_customers(): # needs to del with database
+    return [
+        ("898", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("897", "Name", "Lastname", "Address", "phone", "email", "inactive", 10)
+    ]
+
+@pytest.fixture
+def _update_customer_credit(): # needs to del with database
+    return [
+        ("798", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("797", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("796", "Name", "Lastname", "Address", "phone", "email", "inactive", -99)
+    ]
+
+@pytest.fixture
+def _list_active_customers():
+    return [
+        ("598", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("597", "Name", "Lastname", "Address", "phone", "email", "inactive", 10),
+        ("596", "Name", "Lastname", "Address", "phone", "email", "inactive", 99),
+        ("595", "Name", "Lastname", "Address", "phone", "email", "active", 999),
+        ("594", "Name", "Lastname", "Address", "phone", "email", "active", 10),
+        ("593", "Name", "Lastname", "Address", "phone", "email", "active", 99)
+    ]
+
+def test_list_active_customers(_list_active_customers):
+    """ actives """
+    for customer in _list_active_customers:
+        l.add_customer(customer[0],
+                       customer[1],
+                       customer[2],
+                       customer[3],
+                       customer[4],
+                       customer[5],
+                       customer[6],
+                       customer[7]
+                       )
+    actives = l.list_active_customers()
+
+    assert actives == 4
```

Comment:
I believe this would need to be updated to assert len(actives) == 4 since you are comparing how many active accounts there are